README.markdown

```diff
@@ -6841,6 +6841,7 @@ If a number-like argument is specified (including strings that look like numbers
 If a non-number-like string argument is specified, then it is interpreted as a "pattern". The following patterns are supported:
 
 * `'*a'`: reads from the socket until the connection is closed. No end-of-line translation is performed;
+* `'*b'`: the BSD style receive, return anything once socket receives;
```

Comment:
I think a bit more about this. Maybe rename it to `*p` to mean receiving a "packet" makes more sense? Thoughts?

Reply:
@mike07026 I think maybe it's better to do a `bsdrecv()` method instead.